src/components/TextAreaAutoSize/index.jsx

```diff
@@ -25,6 +25,7 @@ const TextAreaAutoSize = ({
         isValid={touched && !error}
         isInvalid={hasError}
         rows={3}
+        autoResize
```

Comment:
The purpose of `TextAreaAutoSize` is to have the `textarea` element resize itself vertically. Support for this was removed when this component was refactored to use `Form.Control`, which likely did not support the `autoResize` prop at the time.